OutOfSchool/OutOfSchool.AuthCommon/Validators/ICustomPasswordRules.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace OutOfSchool.AuthCommon.Validators;
+public interface ICustomPasswordRules
```

Comment:
(Code style) Please add empty line after namespace line.

Reply:
Fixed. Thank you!